ecs/example/main.tf

```diff
@@ -37,6 +37,24 @@ module "cluster" {
   tags = "${var.tags}"
 }
 
+# ------------------------------------------------------------------------------
```

Comment:
I'm thinking we still need the `ecs/cluster` example in there? Would it work to just stick both in there, use the output from `ecs/cluster` for the other modules, and just write `Option A: ecs/cluser` and `Option B: ecs/spotfleet` over the configurations? (the `---` headers).

Reply:
Copy paste error - full file now has 2 modules with same name (both cluster) will rename the spotfleet one in next push